niworkflows/data/getters.py

```diff
@@ -22,7 +22,8 @@
     'mni152_nlin_sym_las': ('57fa7fc89ad5a101e635eeef', '9c4c0cad2a2e99d6799f01abf4107f5a'),
     'mni152_nlin_sym_ras': ('57fa7fd09ad5a101df35eed0', '65d64ad5a980da86e7d07d95b3ed2ccb'),
     'mni_icbm152_linear': ('580705eb594d9001ed622649', '72be639e92532def7caad75cb4058e83'),
-    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec')
+    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec'),
+    'oasis_dkt31_mni152': ('5a95cd9591b689000ca5b6ab', 'e0bea438b9645c9e7c333a6edd859715'),
```

Comment:
@oesteban, @mgxd - was this regenerated by registering the OASIS template to the mni asym template? or simply used as being in MNI space. if that latter, i would highly recommend running the registration and then moving the atlas over.

Reply:
this was generated by registering the oasis template brainmask to the 2mm mni-asym template, and then applying the transform to the label atlas